render.go

```diff
@@ -247,6 +247,12 @@ func (r *Render) compileTemplatesFromAsset() {
 	}
 }
 
+// Wrapper around template.Lookup(). Returns the template with the given
+// name that is associated with t, or nil if there is no such template
+func (r *Render) Lookup(name string) *template.Template {
```

Comment:
Lets call this `TemplateLookup`. Just so it's not confused with any of the other renderers.